lang/java/reef-runtime-multi/src/main/java/org/apache/reef/runtime/multi/client/MultiRuntimeConfigurationBuilder.java

```diff
@@ -85,11 +86,12 @@ public MultiRuntimeConfigurationBuilder setDefaultRuntime(final String runtimeNa
    * @return The builder instance
    */
   public MultiRuntimeConfigurationBuilder setSubmissionRuntime(final String runtimeName) {
```

Comment:
Please mark the whole class as `@Unstable`. That allows us to follow up on our ideas for a better API when 0.16 comes around.